bin/webpack-configs.js

```diff
@@ -137,6 +153,43 @@ woocommerce_blocks_env = ${ NODE_ENV }
 	};
 };
 
+// This is meant to fix issue #3839 in which we have two instances of SlotFillProvider context. Should be deleted once wordpress/gutenberg#27462.
```

Comment:
maybe add a `@todo` to this line so an issue is created when this PR is merged? That will make sure we don't forget to remove this when it's no longer needed.